### ISSUE TITLE
Can I use this with an SVG as a React Component?

### ISSUE THREAD

**Issue by joaoesp** [BetterTyped/react-zoom-pan-pinch#411]
I am using an SVG as a React Component with svgr:
`import { ReactComponent as Map } from "../assets/world-map.svg";`

Is this supposed to work? I am not seeing anything but it works without the transform elements: 
```
<TransformWrapper>
    <TransformComponent>
        <Map />
    </TransformComponent>
</TransformWrapper>
```

**Comment by jCrip** [comment]
yes, we are currently using this like that

**Comment by prc5** [comment]
Yes — inline SVG React components work as children of `TransformComponent`; the `svg-zoom-to-element` story shows it. Closing as answered.

---
_Part of the September 2026 triage of every open issue against v4.0.7. If this still applies on 4.0.7, reply with a reproduction and we will reopen._
